.mailmap

```diff
@@ -8,6 +8,7 @@
 #
 Aaron Bassett <abassett@gmail.com>
 Abhishek Lekshmanan <abhishek.lekshmanan@ril.com> <abhishek.lekshmanan@gmail.com>
+Abhishek Lekshmanan <abhishek@suse.com> <alekshmanan@suse.com>
```

Comment:
So there is already an entry in peoplemap, so that's not necessary, however one more entry like
`Abhishek Lekshmanan <abhishek@suse.com> <abhishek.lekshmanan@gmail.com>` may be necessary here in .mailmap for affiliating reviews etc.